### PR TITLE
Docs/general update

### PR DESCRIPTION
**Pull request by BigSamu**
Solves issue #38 for updating all documentation on Changelog and Release Notes Process

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+As with other types of contributions, the first step is to [**open an issue on GitHub**](https://github.com/BigSamu/OpenSearch_Parse_Changelog_Action/issues/new). Opening an issue before you make changes makes sure that someone else isn't already working on that particular problem. It also lets us all work together to find the right approach before you spend a bunch of time on a PR. So again, when in doubt, open an issue.
+
+Once you've opened an issue, check out our [Developer Guide](./DEVELOPER_GUIDE.md) for instructions on how to get started.
+
+<p align="right">(<a href="#back-to-top">back to top</a>)</p>
+
+## Developer Certificate of Origin
+
+This **Automated Changelog and Release Notes Process** is licensed under the MIT license (see either the license page at the [Open Source Initiative](https://opensource.org/license/mit/) or the [LICENSE.txt file](./LICENSE.txt)).
```
[the remark this answers is not in the document]
Comment (JohnathonBowers) [reply CONTRIBUTING.md:67]: Made the change!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+**Only submit your own work** (or work you have sufficient rights to submit) - Please make sure that any code or documentation you submit is your work or you have the rights to submit. We respect the intellectual property rights of others, and as part of contributing, we'll ask you to sign your contribution with a "Developer Certificate of Origin" (DCO) that states you have the rights to submit this work and you understand we'll use your contribution. There's more information about this topic in the [DCO section](#developer-certificate-of-origin).
+
+<p align="right">(<a href="#back-to-top">back to top</a>)</p>
+
+## Ways to Contribute
+
+### Join the Discussion
+
+See OpenSearch-Dashboard's [communication guide](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/COMMUNICATIONS.md) for information on how to join our Slack workspace, forum, or developer office hours.
```
[the remark this answers is not in the document]
Comment (JohnathonBowers) [reply CONTRIBUTING.md:33]: Makes sense. I'll remove this section.

**Review by BigSamu — COMMENTED** [review]
Like the changes. Although we should keep consistency for the name of the services, which are:

1. OpenSearch Changelog Workflow
2. OpenSearch Changelog PR Bridge
3. OpenSearch Chnagelog Bot

**Comment by BigSamu** [comment]
@JohnathonBowers,

The initial merge for all documentation has been done. I will open a new PR to continue working on lasting parts of documentation for the README.md file

**Pull request merged**